dojo/settings/settings.dist.py

```diff
@@ -377,6 +377,19 @@ def generate_url(scheme, double_slashes, user, password, host, port, path, param
             'PASSWORD': env('DD_DATABASE_PASSWORD'),
             'HOST': env('DD_DATABASE_HOST'),
             'PORT': env('DD_DATABASE_PORT'),
+            if os.getenv('DD_DATABASE_OPTIONS') is not None:
```

Comment:
Is this valid syntax? To use `if ...:` in the Python dictionary? I suppose it should be defined after the dictionary.

Reply:
It is not valid syntax. I'm going to move to an elif before the dictionary.